### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by Danc2050** [ConsenSys/mythril#1311]
Hello.

Quick question. Is there a way to see in the output of mythril what contract mythril was called on? I did not have enough foresight to know that I needed this and I wondered if it was hidden in the output somewhere.

**Comment by norhh** [comment]
can you elaborate on it? If -v4 might also have the data of which contract Mythril is running on at the moment

**Comment by Danc2050** [comment]
Basically I outputted to a large file some mythril output. Unfortunately, if the contract doesn't have an exploit in itself (rather the exploit lies in a contract it calls), then no output will list the contract name and a parser I created will not be able to determine what contract has the exploit. It is nice that mythril distinguishes what is an exploit from another contract vs an exploit of itself (it does so, intentionally I don't know, by using the checksummed version of a contract to indicate it is the calling contract that has some kind of exploit).

e.g., if I run
`myth -v4 analyze -1 --execution-timeout 3600 -a 0x139132e99C03d9bCE03256796163303603DC1A59` and I only have the output of that execution (not the command I ran), I do not know what contract the output belongs to.

It is my bad of course and I have a way to find out the contract name by brute-forcing a search through looking up the bytecode of a particular range of addresses my machine was dealing with, but its not ideal.
